### PR TITLE
fix incorrect usage of theming

### PR DESCRIPTION
**Pull request by TomaszCz**
To test the changes of course link example to storybook_toolkit 
so you must point like this:
  storybook_toolkit: 
    path: ../../storybook_toolkit
    
Result:
image

And color is settable via canvasColor, version bumped +1

### CODE REVIEW

File: lib/src/storybook.dart [hunk 1]
```diff
@@ -16,41 +16,6 @@ import 'package:storybook_toolkit/src/plugins/text_sizer.dart';
 import 'package:storybook_toolkit/src/plugins/theme/code_view_syntax_theme.dart';
 import 'package:storybook_toolkit/storybook_toolkit.dart';
 
-class _ThemeWrapperBuilder {
-  _ThemeWrapperBuilder();
-
-  static Widget themeBuilder(
```
Comment (mjablecnik) [inline lib/src/storybook.dart:22]: Why did you remove this themeBuilder?

**Comment by mjablecnik** [comment]
@TomaszCz Can you update also CHANGELOG.md?

File: lib/src/storybook.dart (hunk 1 above)
Comment (TomaszCz) [reply lib/src/storybook.dart:22]: No longer used.

I could refactor it and keep e.g. MaterialApp as separate widget ( similar what wrapper did ). To be honest, I see no value in that, the logic got inlined and it looks much more clean.

Comment (mjablecnik) [reply lib/src/storybook.dart:22]: Yes, I understand. I will merge it and refactor the build method of _StorybookState when I have time. But if you have some ideas and time for contribution, you can make another PR ;)

**Comment by TomaszCz** [comment]
Hi! @mjablecnik I see the PR is merged, yay :) how about publishing it on pub.dev ;)
